sle15/profiles/stig.profile

```diff
@@ -147,6 +147,7 @@ selections:
     - dconf_db_up_to_date
     - dconf_gnome_banner_enabled
     - dconf_gnome_login_banner_text
+    - dir_permissions_library_dirs
```

Comment:
There is a conflict on this file that should be addressed.

Reply:
rebased the branch so conflict should be resolved now in 07143f3